### DISCUSSION TITLE
Names for claims

### DISCUSSION THREAD

**Discussion by 1UnderTheSun** [GriefPrevention/GriefPrevention#2231] [Ideas]
Naming your homes exists, but naming your claims does not. I think it'd be a very nice addition for when you /claimslist. Instead of being bombarded with a heap of coordinates, it'd be nice to have a name for them.

**Comment by Jikoo** [comment]
Sounds like you want https://dev.bukkit.org/projects/claimclassifier

**Reply by NumbGnat** [reply]
I came here to propose the same idea. Being able to name claims would be extremely helpful! 💯 

It doesn't look like this project is actively maintained anymore... 😕

Would it be reasonable to request that this functionality be added directly into the main plug-in?

**Reply by Jikoo** [reply]
Focused addons like this can achieve maturity quickly. Because GP does its best not to break compatibility, they are stable without needing updates. Not seeing code changes doesn't mean it doesn't work.

Adding this to GP itself would require database modifications, which is a lot of headache for a relatively niche feature. There is certainly space to discuss it when the database rewrite occurs for v20, but I don't personally believe it has enough use to merit being added as a core feature.
